src/SimpleAmqpClient/Channel.h

```diff
@@ -523,8 +523,9 @@ class SIMPLEAMQPCLIENT_EXPORT Channel : boost::noncopyable {
    * overload
    * doesn't require the Envelope object to Acknowledge
    * @param delivery_info
+   * @param multiple if true, ack all messages up to this delivery tag, if false ack only this delivery tag
```

Comment:
Note that ack'ing multiple message is scoped messages delivered on a given AMQP channel. SimpleAmqpClient uses one channel per consumer, so multiple ack means all un-ack'd messages up to and including the current message id for a given consumer. 

Please make sure to include the above clarification in the documentation added to the header.

Reply:
Done